### PR TITLE
updates

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 language: node_js
 node_js:
   - "0.10"
+  - "4.3"
```
Comment (david-martin) [inline .travis.yml]: this isn't the cause of the problem, but no need for 4.3 here. we can keep 4.4

**Comment by david-martin** [comment]
@matzew The issue seems to be with building the protaganist dep.
https://github.com/feedhenry-templates/fh-connector-sharepoint-cloud/blob/master/package.json#L20
Maybe we should update this dep version if theres a newer version that could work with node 4 & 0.10?

**Comment by matzew** [comment]
@david-martin yeah, I did play with that too

https://github.com/apiaryio/protagonist/tree/v1.3.3

got this error:
https://travis-ci.org/feedhenry-templates/fh-connector-sharepoint-cloud/jobs/139394450#L843

I think I am out of ideas

**Comment by david-martin** [comment]
@matzew lets merge this now that travis passes for node4

**Comment by matzew** [comment]
@david-martin 😘

**Pull request merged**